test/tests_nso_to_oc/xe/xe_network_instance-cgmp.yml

```diff
@@ -0,0 +1,109 @@
+---
+- hosts: localhost
+  gather_facts: no
+  environment:
+    NSO_DEVICE: xe1
+  vars:
+    device: "{{ lookup('env', 'TEST_DEVICE_XEROUTER') | default('xe1', True) }}"
+  tasks:
+    - name: Run XE network instance CGMP interface tests
+      include_role:
+        name: ned-to-oc-test
+      vars:
+        will_load_rollback: False
```

Comment:
This line needs to be deleted, otherwise the device state won't be rolled back. and the next set of test won't have a clean state.